src/main/kotlin/no/nav/familie/tilbake/service/beregning/TilbakekrevingBeregnerVilkår.kt

```diff
@@ -0,0 +1,134 @@
+package no.nav.familie.tilbake.service.beregning
+
+import no.nav.familie.tilbake.common.Periode
+import no.nav.familie.tilbake.domain.tbd.Aktsomhet
+import no.nav.familie.tilbake.domain.tbd.AnnenVurdering
+import no.nav.familie.tilbake.domain.tbd.VilkårsvurderingAktsomhet
+import no.nav.familie.tilbake.domain.tbd.Vilkårsvurderingsperiode
+import no.nav.familie.tilbake.domain.tbd.Vurdering
+import java.math.BigDecimal
+import java.math.RoundingMode
+
+internal object TilbakekrevingBeregnerVilkår {
```

Comment:
Er litt usikker på denne og logikken for ukedager. Sidan ytelsane vi skal støtte er for heile måneder, så tror eg ikkje me trenger same beregningen og ukedager støtte som i fptilbake.